components/tools/OmeroPy/test/integration/clitest/test_tag.py

```diff
@@ -41,11 +44,16 @@ def teardown_method(self, method):
         self.teardown_mock()
 
     @classmethod
+    def new_tag(self, name=""):
```

Comment:
Hard to tell, this could arguably be moved to the upstream library but it is also a 2-line method.